packages/rust.rb

```diff
@@ -32,7 +32,7 @@ def self.install
     ENV['RUSTUP_HOME'] = "#{CREW_DEST_PREFIX}/share/rustup"
     default_host = ARCH == 'aarch64' || ARCH == 'armv7l' ? 'armv7-unknown-linux-gnueabihf' : "#{ARCH}-unknown-linux-gnu"
     downloader 'https://sh.rustup.rs', '173f4881e2de99ba9ad1acb59e65be01b2a44979d83b6ec648d0d22f8654cbce', 'rustup.sh'
```

Comment:
Add to `source_url` and `git_hashtag` instead?

Reply:
Change `system "install -Dm644 ...` to `FileUtils.install ... mode: 0o644`?

Reply:
Sure... changing....